src/index.js

```diff
@@ -67,11 +68,14 @@ async function run(request, context) {
     }
     const [packageName, serviceName] = funcName.split('--');
 
+    // Use mapped subsystem if available, else fallback to default
+    const subsystemMapped = mapSubsystem(alias ?? funcVersion, subsystem, context, log);
+
```

Comment:
instead of passing the defaultSubSystem, I'd use:
```
const subsystem = mapSubsystem(alias ?? funcVersion, context, log) || content.env.CORALOGIX_SUBSYSTEM;
```

Reply:
for reasons of separation of concerns i'd like to keep the determination of subsystem mapping with in the function.

Reply:
I don't agree: `mapSubsystem` can only map a subsystem if there *is* a JSON in the env variable. Otherwise it is **not** his concern to say what the subsystem is. For instance, if there ever is another way of determining a subsystem, we first have to call that, and the pass that value to `mapSubsystem`, which is awkward.

Reply:
makes sense, i have adjusted the PR accordingly.